iocore/net/UnixNetVConnection.cc

```diff
@@ -1352,6 +1352,10 @@ TS_INLINE void
 UnixNetVConnection::set_inactivity_timeout(ink_hrtime timeout_in)
 {
   Debug("socket", "Set inactive timeout=%" PRId64 ", for NetVC=%p", timeout_in, this);
+  if (timeout_in == 0) {
+    // set default inactivity timeout
+    timeout_in = HRTIME_SECONDS(nh->config.default_inactivity_timeout);
```

Comment:
We should add this so we know the default timeout is being applied:
` NET_INCREMENT_DYN_STAT(default_inactivity_timeout_applied_stat);`